servers/rendering/renderer_rd/renderer_scene_gi_rd.h

```diff
@@ -611,9 +611,9 @@ class RendererSceneGIRD {
 		uint32_t blend_ambient;
 		uint32_t texture_slot;
 
-		float anisotropy_strength;
-		float ao;
-		float ao_size;
+		uint32_t pad0;
+		uint32_t pad1;
+		uint32_t pad2;
```

Comment:
Is this just to pad to a specific size? Would be good to add a comment to make it explicit, so we know what to remove if adding more fields and reclaiming the padding.

Reply:
its a push constant for Vulkan, needs to be paded to 16, but there are lots of those like this around the renderer.